src/blueapi/startup/example_plans.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+
+from bluesky.protocols import Movable, Readable
+
+from blueapi.plans import count, move
+
+from ..core import MsgGenerator, inject
```

Comment:
If it's in a different package it's better to use the canonical path

```suggestion
from blueapi.core import MsgGenerator, inject
```